Resources/Prototypes/_Impstation/Entities/Objects/Misc/cards.yml

```diff
@@ -1074,3 +1074,1201 @@
   - type: Sprite
     sprite: _EstacaoPirata/Objects/Misc/cards.rsi
     state: sc_The_World_tarot
+
+# TT cards
+
+- type: entity
+  id: TTRules
```

Comment:
I'm going to politely ask you to change all prototypes with "TT" in the name to TippyTrump so it can be more easily searched